sigle/src/ui/Typography.ts

```diff
@@ -8,7 +8,7 @@ export const Typography = styled('p', {
         lineHeight: '62px',
         letterSpacing: '-0.3px',
 
-        '@xl': {
+        '@md': {
```

Comment:
at the start of the task I merged changes from this pr https://github.com/sigle/sigle/pull/724, where we discussed that it would be better to have the breakpoint at `md` rather than `xl`

Reply:
Okay then does `lg` make more sense?

Reply:
yeah that makes sense :) will update